cha/req.tex

```diff
@@ -18,14 +18,15 @@ \chapter{Requirement Elicitation}\label{cha:requirement_elicitation}
 
         Under normal circumstances each individual device in a session should have control of their own volume and whether they are playing or not, however if the volume affects the desired psychoacoustic effect then the session host may assume control of volume on any device in the session.
     
-    \item\label{req:manipulate} \textbf{Manipulate playback on all devices in a session.} \hfill\\
-        As mentioned in the problem statement in \cref{cha:problem_statement}, the app must be able to manipulate the playback of the audio. 
-        To specify, the session host must be able to manipulate the playback on all devices connected to its session.
+    \item\label{req:manipulate} \textbf{For all devices in a session it should be possible to manipulate the playback on each individual device.} \hfill\\

```

Comment:
"It should be possible to manipulate the playback on each device separately"
Måske?